transformer.py

```diff
@@ -84,7 +92,7 @@ def callback(channel, method, properties, body):
         channel.basic_ack(delivery_tag=method.delivery_tag)
 
 
-def transform_single_file(file_path, output_path):
+def transform_single_file(file_path, output_path, servicex):
```

Comment:
I don't think this will work in command line mode where the user provides a --path argument. I think this should default to `None` and then add some checks wherever you reference this to see if serviceX is defined